plugins/cpp/test/src/cpppropertiesservicetest.cpp

```diff
@@ -88,7 +114,8 @@ TEST_F(CppPropertiesServiceTest, ClassPropertiesTest)
       {"Qualified name", "cc::test::SimpleClass"}
     };
 
-    checkProperties(_simpleClassHeader, 9, 10, expected);
+    checkProperties(_simpleClassHeader, 9, 10, expected,
+                    {"Size"}, {"Alignment"});
```

Comment:
We should consider testing the actual value of "size" property by using the `sizeof` operator.